src/ontology/cl-edit.owl

```diff
@@ -30663,6 +30664,15 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://purl.obolibrary.org/obo/BTO_0005588"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A non-ciliated epithelial cell that is part of some oviduct. It is known in some mammals that a peg cell might express EPCAM, CD44, and ITGA6. in vitro, a peg cell has stem-cell-like qualities, including self-renewal capability.")
```

Comment:
I'd prefer a more complete & descriptive text definition with enough information to be useful to a biologist/clinician trying to align this with their knowledge of the biology and help them to recognise the cell type.  I'd also prefer to keep terms like 'some' in the logic.  There's plenty of material to work with in the paper:

<img width="960" alt="image" src="https://user-images.githubusercontent.com/112839/216593032-3f9db5ea-368f-4a85-947e-469a5a36b294.png">

<img width="962" alt="image" src="https://user-images.githubusercontent.com/112839/216594847-0fcee7dc-08e0-423b-b52a-9fd1de4fdd14.png">

Suggested text:

Def: A small, peg-like epithelial cell of the fallopian tube (oviduct) epithelium, with very little cytoplasm.  These cells are much rarer than the ciliated and secretory epithelial cells of the fallopian tube epithelium and are often found intercalated between them, with high concentrations at the fimbriated, distal end of the tube.  They may have a regenerative/stem-cell function.  In humans, markers include EPCAM, CD44, and ITGA6.

comment: In-vitro evidence supports a stem-cell/regenerative function for these cells (PMID:22911892)

Note - I try to keep evidence in the comments & assertions in the definition.  Less certain information and species specific markers are at the end of the definition.

Reply:
Thank you for the suggestions!